grpc-bom/pom.xml

```diff
@@ -25,15 +25,11 @@
     <parent>
         <groupId>dev.resteasy.grpc</groupId>
         <artifactId>resteasy-grpc-parent</artifactId>
-        <version>1.0.0.Alpha6-SNAPSHOT</version>
+        <version>1.0.0.Alpha6</version>
```

Comment:
Is there a reason for removing the SNAPSHOT? I feel like this commit should be left off the PR and done separately after this PR is merged.

Reply:
I guess I jumped the gun there ...